packages/block-library/src/navigation/edit.js

```diff
@@ -146,6 +128,8 @@ function Navigation( {
 		selectBlock( clientId );
 	}
 
+	const hasPages = hasResolvedPages && pages && pages.length;
```

Comment:
can we use the `?.` optional chaining in this repo?

Reply:
Yes but I'd really like this PR to stay as a kind of "revert" to avoid further confusion. I can make this item on my todo in my other PR https://github.com/WordPress/gutenberg/pull/18869 if that's ok?